app/javascript/components/recipes/new/search-ingredient-list.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="w-96">
+    <div class="flex space-x-4">
+      <input
+        :placeholder="$t(`msg.ingredients.search`)"
+        class="w-full mx-1 my-4 p-2 border-2 rounded border-gray-300"
+      />
+    </div>
+    <table class="min-w-full divide-y divide-gray-200 border-2">
+      <thead class="justify-between">
+        <tr class="bg-gray-100  py-1">
+          <th 
+            class="border-solid border-2 border-color-gray-300"
+            v-for="attribute in header"
+            :key="attribute"
+          >
+            <span class="text-black font-bold">{{ $t(`msg.ingredients.${attribute}`) }}</span>
+          </th>
+          <th/>
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected a space before '/>', but not found                                  vue/html-closing-bracket-spacing